sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -138,6 +140,15 @@ object GpuHashJoin {
     }
   }
 
+  // Check whether the entire tree is ast-able or being able to split non-Ast-able conditions
```

Comment:
Nit: would be nice to be consistent with ast capitalization.
```suggestion
  // Check whether the entire tree is ast-able or being able to split non-ast-able conditions
```